packages/uiweb/src/lib/components/chatAndNotification/MinimisedModalHeader.tsx

```diff
@@ -96,8 +77,21 @@ export const MessageBoxHeader = () => {
       setSearchedChats(null);
     }
   };
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const result = await resolveWeb3Name(selectedChat?.did, env);
+        if (result) setWeb3Name(result);
+      } catch (e) {
+        //console.debug(e);
+      }
+    })();
+  });
```

Comment:
```suggestion
  useEffect(async () => {
   try {
        const result = await resolveWeb3Name(selectedChat?.did, env);
        if (result) setWeb3Name(result);
      } catch (e) {
        //console.debug(e);
      }
  },[]);
```

Reply:
<img width="753" alt="image" src="https://github.com/user-attachments/assets/a35091ba-56d5-4cee-9e01-0edf1c2083fa">
@rohitmalhotra1420  using this gives  the above error

Reply:
Let's try this approach.

```
const useName = () => {
  const [data, setData] = useState();

  const getData = useCallback(async () => {
    // call your name resolving function
  });

  useEffect(() => {
    getData();
  });

  return { data };
};

```